database/seeders/CustomerSeeder.php

```diff
@@ -0,0 +1,136 @@
+            'id' => 2,
+            'username' =>'metatrainer',
+            'first_name'=>'Meta',
+            'last_name'=>'Trainer',
+            'email'=>'metatrainer@dev.denhac.org',
+            'member'=>true,
+            'id_checked' => true
+        ]);
+
+        $ops = Customer::factory()->create([
+            'id' => 3,
+            'username' =>'ops',
+            'first_name'=>'Ops',
+            'last_name'=>'Manager',
+            'email'=>'ops@dev.denhac.org',
+            'member'=>true,
+            'id_checked' => true
+        ]);
+        
+        $woodtrainer = Customer::factory()->create([
+            'id' => 4,
+            'username' =>'woodtrainer',
+            'first_name'=>'Woodshop',
+            'last_name'=>'Trainer',
+            'email'=>'wood@dev.denhac.org',
+            'member'=>true,
+            'id_checked' => true
+        ]);
+
+        $textilestrainer = Customer::factory()->create([
+            'id' => 5,
+            'username' =>'textilestrainer',
+            'first_name'=>'Textiles',
+            'last_name'=>'Trainer',
+            'email'=>'fabric@dev.denhac.org',
+            'member'=>true,
+            'id_checked' => true
+        ]);
+
+        Customer::factory()->create([
+            'id' => 6,
+            'username' =>'member',
+            'first_name'=>'Regular',
+            'last_name'=>'User',
+            'email'=>'a.member@dev.denhac.org',
+            'member'=>true,
+            'id_checked' => true
+        ]);
+
+        Customer::factory()->create([
+            'id' => 7,
+            'username' =>'lurker',
+            'first_name'=>'Inactive',
+            'last_name'=>'User',
+            'email'=>'not.a.member@dev.denhac.org',
+            'member'=>false,
+            'id_checked' => true
+        ]);
+
+        Customer::factory()->count(100)->create();
+
+        $memberships = [
+            [$superuser, [
+                UserMembership::MEMBERSHIP_FULL_MEMBER,
+                UserMembership::MEMBERSHIP_BOARD, 
+                UserMembership::MEMBERSHIP_OPS_MANAGER, 
+                UserMembership::MEMBERSHIP_SAFETY_MANAGER, 
+                UserMembership::MEMBERSHIP_BUSINESS_MANAGER,
+                UserMembership::MEMBERSHIP_EVENTS_MANAGER,
+                UserMembership::MEMBERSHIP_TREASURER,
+                UserMembership::MEMBERSHIP_3DP_TRAINER,
+                UserMembership::MEMBERSHIP_3DP_USER,
+                // Magic numbers from TrainableEquipmentSeeder
+                201,202,203,204,205,206,
+                301,302,303,304,305,306
+            ]],
+            [$boarduser, [UserMembership::MEMBERSHIP_BOARD]],
+            [$metatrainer, [UserMembership::MEMBERSHIP_META_TRAINER]],
+            [$ops, [UserMembership::MEMBERSHIP_OPS_MANAGER]],
+            [$woodtrainer, [201,202,203,204,301,302,303,304]],
+            [$textilestrainer, [205, 305]]
+
+        ];
+
+        foreach($memberships as [$customerId, $planIds]) {
```

Comment:
fwiw, this isn't a customer id but it's a full blown customer. Laravel will translate the model to the id for you.

Reply:
That's what I expected too, but when I used it as a customer object it failed, and subsequent logging suggested it was just returning an ID. IDK what I was doing wrong but this works. How do you recommend we change it?